cassandra-mesos-scheduler/src/test/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/api/ClusterBackupControllerTest.java

```diff
@@ -0,0 +1,91 @@
+/**
+ *    Copyright (C) 2015 Mesosphere, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.mesosphere.mesos.frameworks.cassandra.scheduler.api;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.mesosphere.mesos.util.Tuple2;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ClusterBackupControllerTest extends AbstractApiControllerTest {
+
+    @Test
+    public void testBackup() throws Exception {
```

Comment:
Add another test when the default value for name has been added to make sure that the parameter is optional.